optimize/apis-tools/optimize-api/tutorial.md

```diff
@@ -0,0 +1,167 @@
+
+```
+  const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+6. Call the collection's endpoint, process the results from the API call, emit the dashboard IDs to output, and emit an error message from the server if necessary:
+
+```
+  try {
+    const response = await axios(options);
+    const results = response.data;
+
+    results.forEach(x => console.log(`ID: ${x.id}`));
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
+
+7. In your terminal, run `node cli.js optimize list` for a list of your existing dashboard IDs. If you have any existing dashboards within a collection, you will see an output similar to the following:
+
+`ID: 12345`
+
+:::note
+This `list` command is connected to the `listDashboards` function at the bottom of the `optimize.js` file, and executed by the `cli.js` file. While we will view dashboard IDs and delete a dashboard in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+If you have any existing dashboards, the `ID: ${x.id}` will now output. If you have an invalid API name or action name, or no arguments provided, or improper/insufficient credentials configured, an error message will output as outlined in the `cli.js` file.
+
+## DELETE a dashboard
+
+To delete a dashboard, take the following steps:
+
+1. Outline your function, similar to the steps above. Note that the URL endpoint will look different, as you are accessing a different endpoint in this request than in the prior request:
+
+```
+async function deleteDashboard([dashboardId]) {
+console.log(`deleting dashboard ${dashboardId}`);
+
+const optimizeAudience = process.env.OPTIMIZE_AUDIENCE;
+const accessToken = await getAccessToken("components", optimizeAudience);
+const optimizeApiUrl = process.env.OPTIMIZE_API_URL;
+
+const url = `${optimizeApiUrl}/public/dashboard/${dashboardId}`;
+```
+
+2. Configure the API call using the DELETE method:
+
+```
+  const options = {
+    method: "DELETE",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+3. Process the results from the API call. For example:
+
+```
+  try {
+    // Call the delete endpoint.
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    if (response.status === 204) {
+      console.log(`Dashboard ${clientId} was deleted!`);
+    } else {
+      // Emit an unexpected error message.
+      console.error("Unable to delete dashboard!");
+    }
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+}
+```
+
+4. In your terminal, run `node cli.js optimize delete` to delete your dashboard. You will see a response similar to the following:
```

Comment:
Was happy to see an error here if I don't provide a `dashboardId`. But after I did, I'm still getting an ECONNREFUSED when I run the completed file. 

If I step through what's written here, it doesn't work, but I also don't get an error.

I incorporated the changes @pepopowitz made in https://github.com/camunda/camunda-api-tutorials/pull/7/files (adding `/api` to the URL changing the variable in the output to dashboardId). I don't get an error.

Reply:
Okay -- I haven't analyzed the changes from @pepopowitz with the new auth layout into this PR. I was anticipating launching a new PR with the updated auth changes across administration, optimize, and modeler.

I'm thinking it might be best to move this back into `draft`, and I can adjust the auth references across Administration and Optimize in this PR, with a fresh PR for Modeler.

Reply:
There are several bug fixes that I snuck into https://github.com/camunda/camunda-api-tutorials/pull/7, which I should have opened a separate PR for. They were all related to the `delete` functionality.